src/rpc/blockchain.cpp

```diff
@@ -436,15 +436,15 @@ UniValue getrawmempool(const UniValue& params, bool fHelp)
         );
 
     bool fVerbose = false;
-    if (params.size() > 0)
-        fVerbose = params[0].get_bool();
+    if (request.params.size() > 0)
+        fVerbose = request.params[0].get_bool();
 
     return mempoolToJSON(fVerbose);
 }
 
-UniValue getmempoolancestors(const UniValue& params, bool fHelp)
+UniValue getmempoolancestors(const JSONRequest& request)
 {
-    if (fHelp || params.size() < 1 || params.size() > 2) {
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
```

Comment:
I have considered this. But if we would have done this from the beginning, wouldn't it be without the extra `UniValue params` delegation?
I think this PRs solution is cleaner.

We have to do this once and therefore probably accept a larger diff.